_sass/blocks/_header.scss

```diff
@@ -86,6 +125,10 @@ header,
   &:active {
     text-decoration: underline;
   }
+
+  @include respond-to(large-up) {
```

Comment:
Expected item on line 129 to appear before line 124. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets